tools/ci.sh

```diff
@@ -749,7 +749,7 @@ ZEPHYR_SDK_VERSION=0.16.8
 ZEPHYR_VERSION=v3.7.0
 
 function ci_zephyr_setup {
-    IMAGE=zephyrprojectrtos/ci:${ZEPHYR_DOCKER_VERSION}
+    IMAGE=ghcr.io/zephyrproject-rtos/ci:${ZEPHYR_DOCKER_VERSION}
```

Comment:
Is this mirror kept up-to-date?

Reply:
Should be, both registries are published by the official Zephyr project and they link to both in their docs:
https://github.com/zephyrproject-rtos/docker-image/?tab=readme-ov-file#using-pre-built-developer-docker-image

Reply:
I see, great!